bandit/README.md

```diff
@@ -0,0 +1,23 @@
+## support-bandit
+
+Defines three lambdas `get-bandit-tests`, `query-lambda` and `calculate-lambda` that form a state machine (using Step Functions) to record view and acquisition data for multi-armed bandit experiments. RRCP channel tests (only Epics for now) that are marked as bandit experiments will be picked up and their performance data written to a DynamoDB table. The state machine is scheduled to run once an hour.
```

Comment:
Could we add a link here to a good source to explain what multi-armed bandit testing actually is?